src/forms/CreateCircleForm.tsx

```diff
@@ -0,0 +1,50 @@
+import { z } from 'zod';
+
+import { createForm } from './createForm';
+
+const schema = z
+  .object({
+    user_name: z
+      .string()
+      .min(3, 'User name must be at least 3 characters long.'),
+    circle_name: z
+      .string()
+      .min(3, 'Circle name must be at least 3 characters long.'),
+    protocol_name: z
+      .string()
+      .min(3, 'User name must be at least 3 characters long.'),
```

Comment:
spelling: "Protocol" not user here.

Reply:
Organization, in fact 😅  We want to avoid using the name "protocol" in user-facing contexts, it's a legacy naming choice